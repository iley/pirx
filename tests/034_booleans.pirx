extern func main(): int {
  var flag: bool = 1 < 2;
  if flag {
    printf("1 < 2 is true\n");
  }
  while flag {
    printf("inside loop\n");
    flag = !flag;
  }
  return 0;
}
