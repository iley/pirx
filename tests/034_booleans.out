1 < 2 is true
inside loop
